### PR TITLE
bump minimum panel version to 1.3.8

### PR DESCRIPTION
**Pull request by pmeier**

### CODE REVIEW

File: ragna/deploy/_ui/central_view.py [hunk 1]
```diff
@@ -447,34 +447,29 @@ def chat_interface(self):
                     ),
                 )
             ],
+            card_params=dict(
+                stylesheets=ui.stylesheets(
```
Comment (pmeier) [inline ragna/deploy/_ui/central_view.py:451]: This is not going to work. See holoviz/panel#6240. This was "promised" in https://github.com/Quansight/ragna/pull/261#discussion_r1434489553 although the comment might also refer to making `card_params` usable in the first place. cc @ahuang11

**Comment by pmeier** [comment]
As of `1.3.7rc2` there two issues that prevent adoption:

- [ ] Our auth workflow is broken. This happens because for some reason the cookie we set does not show up in `pn.state.cookies`. However, I can confirm that the cookie is indeed there through the browser console. Plus, through randomly refreshing the page and clicking login again I sometimes get past the authentication, i.e. `panel` picks up on the cookie. So far I haven't found pattern.
- [ ] When trying to send a message, we are instantly met with an error before the message is even rendered. Here is the traceback:

    ```
	Traceback (most recent call last):
	  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/panel/chat/feed.py", line 486, in _prepare_response
	    self._callback_state = CallbackState.RUNNING
	  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/contextlib.py", line 126, in __exit__
	    next(self.gen)
	  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/param/parameterized.py", line 307, in batch_call_watchers
	    parameterized.param._batch_call_watchers()
	  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/param/parameterized.py", line 2506, in _batch_call_watchers
	    self_._execute_watcher(watcher, events)
	  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/param/parameterized.py", line 2468, in _execute_watcher
	    watcher.fn(*args, **kwargs)
	  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/panel/reactive.py", line 374, in _param_change
	    self._apply_update(named_events, properties, model, ref)
	  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/panel/reactive.py", line 302, in _apply_update
	    self._update_model(events, msg, root, model, doc, comm)
	  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/panel/layout/base.py", line 119, in _update_model
	    super()._update_model(events, msg, root, model, doc, comm)
	  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/panel/reactive.py", line 623, in _update_model
	    super()._update_model(events, msg, root, model, doc, comm)
	  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/panel/reactive.py", line 318, in _update_model
	    model_val = getattr(model, attr)
	  File "/home/philip/.conda/envs/ragna-dev/lib/python3.9/site-packages/bokeh/core/has_props.py", line 357, in __getattr__
	    return super().__getattribute__(name)
	AttributeError: 'Card' object has no attribute '_callback_state'
    ```

    There is no mention of Ragna at all in the traceback so I guess this is coming from `panel`. I'll try to compile a minimal repro.

File: ragna/deploy/_ui/central_view.py (hunk 1 above)
Comment (philippjfr) [reply ragna/deploy/_ui/central_view.py]: Fixed in panel 1.3.7rc3

**Comment by philippjfr** [comment]
I was not able to track down any changes that could possibly have broken the auth flow. The `panel/auth` module was barely touched and the same is true of any server related code.

**Comment by pmeier** [comment]
> The `panel/auth` module was barely touched and the same is true of any server related code.

Unfortunately, we are rolling our custom auth flow. There is a refactor planned (#178), but for now we are stuck with it.

Here is the idea:

1. When hitting `/` we check if we find a cookie. If that is not the case, we redirect to `/auth`
   
   https://github.com/Quansight/ragna/blob/61e8d5f6618d0533a106e5c27c350e2a2e71621e/ragna/deploy/_ui/app.py#L66-L67
2. On `/auth` we let the user sign in and if that works, we set the cookie and redirect to `/`

   https://github.com/Quansight/ragna/blob/61e8d5f6618d0533a106e5c27c350e2a2e71621e/ragna/deploy/_ui/auth_page.py#L50-L64

So far this works fine with `1.3.7`. However when we hit `/` the second time, `pn.state.cookies` still does not have the cookie we have set. But I can verify through the browser console (`document.cookie`) that the cookie is there.

So it seems however `panel` gets the cookies from the browser is somehow broken. Maybe there was a cache introduced or the like?

**Comment by pmeier** [comment]
Here is the issue that we need fixed for our auth flow: holoviz/panel#6252

**Comment by pmeier** [comment]
Fix for the cookie issue is available in holoviz/panel#6254. However, `1.3.7` was already released and thus we now need to wait for `1.3.8` before we can update.

**Review by nenb — APPROVED** [review]
LGTM!

**Pull request merged**